### PR TITLE
activity selection in ruby added

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Implemented the same algorithm for the activity selection in Ruby language.
issue #531

### CODE REVIEW

**Comment by jainaman224** [comment]
Please refer the issue you are solving.

File: Activity_Selection/Activity_Selection.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  i = 0
+  print (i.to_s + "\t")
+  for j in 1..n-1
+    if start[j]>finish[i]
+      print (j.to_s + "\t")
+      i = j
+    end
+  end
+end
+print_activities(start, finish , n)
```
Comment (jainaman224) [inline Activity_Selection/Activity_Selection.rb]: Please add sample input output.
Then it is ready to be merged.

**Comment by jainaman224** [comment]
@AnshitaVishwa Are you still working on it?

**Comment by AnshitaVishwa** [comment]
sir i have the respective changes ....that u had asked me to do. kindly review it.

**Pull request merged**